Dockerfile

```diff
@@ -2,11 +2,12 @@ FROM python:3.6
 MAINTAINER Zach White <skullydazed@gmail.com>
 
 WORKDIR /
-RUN git clone https://github.com/qmk/qmk_compiler_api.git
-WORKDIR /qmk_compiler_api
-RUN pip3 install git+git://github.com/qmk/qmk_compiler_worker.git@master
-RUN pip3 install git+git://github.com/skullydazed/kle2xy.git@master
+#RUN git clone https://github.com/qmk/qmk_api.git
+WORKDIR /qmk_api
+COPY . /qmk_api
```

Comment:
I'd replace this with `COPY . .`

Reply:
Even if you can use a relative URL there I think it's more clear what is going on with the absolute path for the container argument.

Reply:
Yeah, at least in my case what was being returned was nothing and an exception was being raised. Lol

Reply:
In my experience, it's an established convention to do `COPY . .` if you're copying right after setting the workdir, because then you only need to modify one line to change which directory you're using — and it's still readable and clear since the lines are right next to each other.

Reply:
Yeah that seems reasonable. I might end up tweaking this a bit more though so that I can have an editable copy of the workers as well (which could also tie into https://github.com/qmk/qmk_api/pull/30#discussion_r221191826 ).

Reply:
It may be an established convention, but "Explicit is better than implicit." This is a case where I'd rather be a little more explicit.

Reply:
That would be outside of this Dockerfile's scope, though, since the workers are qmk_compiler territory (which should have its own Dockerfile and docker-compose.yml — we can discuss this in Discord later tonight or tomorrow).

Reply:
@skullydazed I'm okay with either, but imo
```Dockerfile
WORKDIR /qmk_api
COPY . .
```
is explicit enough. Once and Only Once applies here.

Reply:
Honestly, we should not be typing WORKDIR and COPY together in this way in the first place. IMO there should be one WORKDIR statement, maybe at the end just before CMD. Everything else should not depend on what your current working directory is.

Reply:
That's okay too. As long as the host's local PWD is copied into a known path in the container (this is what Compose relies on), and the same path is set as the container's PWD at some point, the ordering doesn't matter much. The suggestion was for how it's most commonly done in similar (Flask and Flask-like) projects.

Honestly, looking at it now, it's bikeshedding. Feel free to accept or disregard the change.